src/prototype_room_init.js

```diff
@@ -286,6 +310,8 @@ Room.prototype.setup = function() {
   });
   let paths_sorted = _.sortBy(paths_controller, sorter);
   let path = this.getMemoryPath(paths_sorted[paths_sorted.length - 1].name);
+  let pathLB = this.getMemoryPath(paths_controller[4].name);
```

Comment:
If you have a `!= 2` sources room it will not fit.

I tried to stay independent from as much as possible.